### PR TITLE
chore(deps): update to 1.22.2

### PR DESCRIPTION
**Pull request by schultzp2020**
## Description

- Updates the showcase to 1.22.2
- Fixes the `version:all` script
- Updates the renovate config

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-902

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]
seems reasonable, and after this we'll get more dependabot/renovatebot updates for dist-dynamic folders. didn't look carefully at `update-backstge.mks`

**Comment by schultzp2020** [comment]
> seems reasonable, and after this we'll get more dependabot/renovatebot updates for dist-dynamic folders. didn't look carefully at `update-backstge.mks`

It should still ignore everything under `dist-dynamic`

**Comment by nickboldt** [comment]
> It should still ignore everything under `dist-dynamic`

You sure? https://github.com/janus-idp/backstage-showcase/pull/938/files#diff-f82f7d36a61d22f640c25bdb8b0435bf9cb38679d99d5f460753fa668ee7cdb3L29-R30

but you specifically removed that from the ignorePaths. What am I missing?

**Comment by schultzp2020** [comment]
> but you specifically removed that from the ignorePaths. What am I missing?

My changes will ignore anything under that path as well, which includes `dist-dynamic`.

File: .github/renovate.json [hunk 1]
```diff
@@ -20,11 +20,12 @@
...
-        "^@roadiehq/"
+        "^@roadiehq/",
+        "^@pagerduty/"
       ],
       "groupName": ["Backstage packages"],
       "dependencyDashboardApproval": true
     }
   ],
-  "ignorePaths": ["**/dist-dynamic/**"]
+  "ignorePaths": ["**/dynamic-plugins/**"]
```
Comment (davidfestal) [inline .github/renovate.json:30]: Why this change ?

Comment (schultzp2020) [reply .github/renovate.json:30]: The plugin folder will be updated manually through the backstage `versions:bump` cli. Currently renovate is creating PRs for dependencies we cannot update in `dynamic-plugins`.

Comment (davidfestal) [reply .github/renovate.json:30]: > dependencies we cannot update in dynamic-plugins.

Can you precise which ones ?

Comment (schultzp2020) [reply .github/renovate.json]: Hmmm, thinking about it more I guess we don't need the change. Reverting...
